src/mixins/canvas_events.mixin.js

```diff
@@ -50,11 +50,12 @@
     },
 
     addOrRemove: function(functor, eventjsFunctor) {
+      var eventTypePrefix = this.get('enablePointerEvents') ? 'pointer' : 'mouse';
       functor(fabric.window, 'resize', this._onResize);
-      functor(this.upperCanvasEl, 'mousedown', this._onMouseDown);
-      functor(this.upperCanvasEl, 'mousemove', this._onMouseMove, addEventOptions);
-      functor(this.upperCanvasEl, 'mouseout', this._onMouseOut);
-      functor(this.upperCanvasEl, 'mouseenter', this._onMouseEnter);
+      functor(this.upperCanvasEl, eventTypePrefix + 'down', this._onMouseDown);
```

Comment:
can we do `var canvasElement = this.upperCanvasEl`  and then call all those functions with `canvasEement` only? this will help minification reduce the code.

Reply:
like this? 2b24cb8322a9d9379da6f1a4b3879413a174f8b7